documentation/rosetta-modelling-component.md

```diff
@@ -1646,14 +1660,15 @@ xField string (1..1);
 A set when mapping is used to set an attribute to a value derived from the input document if a given when clause is met -  e.g. :
 
 ```
-execution Execution (0..1) <"The execution ...">;
-  [synonym CME_SubmissionIRS_1_0 value TrdCaptRpt set when "TrdCaptRpt.VenuTyp" exists]
+alternativeCombustible FuelEnum (1..1)
+  [synonym CONDITIONAL_SET_TO_EXAMPLE_7 set to FuelEnum -> ELECTRICITY when "engineDetail->fuelDetail->alternativeCombustible" exists]
 ```
 
 A set when synonym can include a default to set an attribute to a constant value when no other value was applicable - e.g. :
 
 ```
-[synonym Bank_A value e path "b.c" default to "DEFAULT"]
+capacityUnit string (0..1)
+  [synonym CONDITIONAL_DEFAULT_EXAMPLE_1 value "volumeCapacityUnit" path "engineType->engineDetail" default to "UK Gallon"]
```

Comment:
value volumeCapacityUnity (with no use of bracket)

Reply:
I do not think so, with the brackets, are you saying that [..value "volumeCapacityUnit"..] does not go with the double quotes? if i put [..value volumeCapacityUnit..] without it there is a syntax error.

Reply:
this synonym CONDITIONAL_DEFAULT_EXAMPLE_1 was made already and this example should be good